src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -327,6 +327,10 @@ public function build() {
     ]);
 
     $this->buildCopy();
+    $this->invokeCommands([
+      'blt:drush:redispatch',
```

Comment:
I'm surprised this doesn't need to run after `composer install`. Maybe it doesn't matter?